servlet/jakarta/build.gradle

```diff
@@ -7,23 +7,24 @@ description = "gRPC: Jakarta Servlet"
 
 // Set up classpaths and source directories for different servlet tests
 sourceSets {
-    undertowTest {
```

Comment:
It's unfortunate not to run any of the tests on Java 8, even when there is an implementation that supported it. Is tomcat/undertow completely not supporting Java 8 any more? From the Tomcat website, they don't even have a download link for 10.0 and the last 10.0 was from 2022. Undertow looks to still support Java 8, with 2.2.32 being recent. Could we keep Undertow still working with Java 8?

Reply:
Undertow 2.2.32 does indeed support Java 8, and is still being tested in :grpc-servlet. This test is for undertow 2.3.13, which uses the jakarta.servlet packages, and does not support Java 8 (including even some Deprecated annotations with forRemoval/etc set, which prevents the test from even compiling using `-PfailOnWarnings=true`).

Reply:
Oh, I was thrown off by `undertow-servlet-jakartaee9` using 2.2. Okay, fair enough.